cli/pkg/local/app.go

```diff
@@ -308,6 +309,17 @@ func (a *App) Serve(httpPort, grpcPort int, enableUI, openBrowser, readonly bool
 	gctx := graceful.WithCancelOnTerminate(a.Context)
 	group, ctx := errgroup.WithContext(gctx)
 
+	// Get the latest version available
+	latestVersion, err := update.LatestVersion(gctx)
+	if err != nil {
+		a.Logger.Warnf("error finding latest version: %v", err)
+	}
+
+	versionInfo := &versionInfo{
+		CurrentVersion: a.Version.Number,
+		LatestVersion:  latestVersion.Version,
+	}
```

Comment:
Run this inside the handler func in `versionHandler` so it doesn't add latency to the startup